Swiften/Roster/XMPPRosterImpl.cpp

```diff
@@ -40,6 +40,15 @@ void XMPPRosterImpl::clear() {
     onRosterCleared();
 }
 
+bool XMPPRosterImpl::isMIXChannel(const JID& jid) {
+    std::map<JID, XMPPRosterItem>::const_iterator i = entries_.find(jid.toBare());
```

Comment:
Assuming you really need the explicit const iterator here, I would prefer RosterMap::const_iterator instead.